src/server/scripts/Kalimdor/TempleOfAhnQiraj/instance_temple_of_ahnqiraj.cpp

```diff
@@ -54,27 +47,23 @@ class instance_temple_of_ahnqiraj : public InstanceMapScript
         //If Vem is dead...
         bool IsBossDied[3];
 
-        //Storing Skeram, Vem and Kri.
         ObjectGuid SkeramGUID;
         ObjectGuid VemGUID;
         ObjectGuid KriGUID;
         ObjectGuid YaujGUID;
+        ObjectGuid HuhuranGUID;
```

Comment:
Why is this added? It's not used anywhere (with the code below)

Reply:
@Nefertumm boss_huhuran(Creature* creature) : BossAI(creature, DATA_HUHURAN) { }

:'v

Reply:
In that case, it's not needed